blossom/templates/app/transcribing.html

```diff
@@ -0,0 +1,60 @@
+{% extends "website/partials/base.partial" %}
+{% load static %}
+
+
+{% block content %}
+    {% if source_url %}
+        <div class="row">
+            <div class="col-lg-6">
+                <p class="text-muted fs-6">Pinch, drag, or click around this image to make it easier to read!</p>
```

Comment:
This page seems to be the same as `blossom/templates/app/index.html`, is it possible to de-duplicate these?

Reply:
yep yep will happen. This page replaces index.html